.github/workflows/street_easy.yml

```diff
@@ -13,6 +13,7 @@ jobs:
       AWS_S3_ENDPOINT: ${{ secrets.AWS_S3_ENDPOINT }}
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      STREET_EASY_URL: ${{ secrets.STREET_EASY_URL }}
```

Comment:
we are actually saving all the environmental variables in a .env file that gets pulled from s3 every time we run it, so we don't really need thi

Reply:
I modified the s3 env file and will remove this line